chains/orchestrator-paras/runtime/flashbox/src/tests/common/mod.rs

```diff
@@ -269,6 +269,8 @@ pub fn set_parachain_inherent_data(mock_inherent_data: MockInherentData) {
         relay_chain_state,
         downward_messages: Default::default(),
         horizontal_messages: Default::default(),
+        relay_parent_descendants: Default::default(),
```

Comment:
does not this require a migration?

Reply:
No.

This is inherent data, the runtime accepts both versions and converts them to the latest one. The conversion is done here: https://github.com/paritytech/polkadot-sdk/blob/0bde5c88f6d5570610018e6d3b6ff3d74604b34f/cumulus/pallets/parachain-system/src/lib.rs#L956

Since we have `type RelayParentOffset = 0`, this field is unused.

And the `collator_peer_id` is also unused, it has been added for a future feature but not implemented yet.